torchvision/prototype/datapoints/_image.py

```diff
@@ -58,6 +58,8 @@ def _from_tensor_shape(shape: List[int]) -> ColorSpace:
 
 
 class Image(Datapoint):
+    # For now, this is somewhat redundant with number of channels.
+    # TODO: decide whether we want to keep it?
```

Comment:
For context, this was designed with the use case of other color spaces with the same number of channels. For example, #4029. Since we can always add this later on, I see no harm of removing it now. That being said, I would keep the `ColorSpace` enum for

https://github.com/pytorch/vision/blob/035d99f1e7f7206a8708e76f4568aac0c3b1346d/torchvision/prototype/transforms/functional/_meta.py#L331

which is a generalized replacement for

https://github.com/pytorch/vision/blob/035d99f1e7f7206a8708e76f4568aac0c3b1346d/torchvision/prototype/transforms/functional/_deprecated.py#L12

and

https://github.com/pytorch/vision/blob/035d99f1e7f7206a8708e76f4568aac0c3b1346d/torchvision/prototype/transforms/functional/_deprecated.py#L25